corehq/apps/app_manager/views/forms.py

```diff
@@ -990,7 +990,13 @@ def _form_name(module, form):
 @require_can_edit_apps
 def get_form_datums(request, domain, app_id):
     form_id = request.GET.get('form_id')
-    datums = _get_form_datums(domain, app_id, form_id)
+    try:
+        datums = _get_form_datums(domain, app_id, form_id)
+    except Exception:
+        notify_exception(request, "Error fetching form datums", details={
+            "domain": domain, "app_id": app_id, "form_id": form_id
+        })
```

Comment:
What's this for, just to add additional info to the error report?  I assumed the exception would show up in sentry anyways.

Reply:
The errors here are either Http404 or Http400 which Sentry won't capture. If there is another error then yes, we'll get a duplicate in Sentry.